client/src/components/advancedSelectWidget/MultiTypeAdvancedSelectComponent.js

```diff
@@ -214,6 +214,7 @@ const MultiTypeAdvancedSelectComponent = ({
         placeholder={searchPlaceholder}
         value={value}
         showEmbedded
+        reset={true}
```

Comment:
A boolean prop like this with value `true` can simply be specified as `reset` (there's a lint rule for this [`react/jsx-boolean-value`], but it's not enabled). However, the issue only applies when the user has multiple entityTypes to choose from, so the value should probably be derived from that, or passed down from the place where the component is used. In any case, the current code changes don't achieve the intended goal, since when I type e.g. "ANET", then change the entityType From Reports to People, the search text stays there, but no search results are shown anymore, and as soon as I click in the input field to fire the search, the search text is cleared.